tests/Fixtures/TestBundle/Entity/UserResource.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Metadata\Post;
+use ApiPlatform\Tests\Fixtures\TestBundle\Dto\UserResetPasswordDto;
+use Symfony\Component\Validator\Constraints as Assert;
+
+#[ApiResource()]
+#[Post(
+    uriTemplate: '/user-reset-password',
+    shortName: 'user-reset-password',
+    validate: false,
```

Comment:
we could check that the validation works on Dtos

Reply:
Done in commit a8a7887ad06b08b53309950cacd5d8d25beab716